### PR TITLE
Docs: minor amendments to runpy.rst

### PR DESCRIPTION
**Pull request by geryogam**
This PR will:

- fix the typo in `mod_name + '.__main__`;
- fix punctuation;
- fix the formatting of some parameters;
- add missing possessive forms;
- use the parameter `file_path`;
- ~use the verb “execute” consistently.~

### CODE REVIEW

File: Doc/library/runpy.rst [hunk 1]
```diff
@@ -113,26 +113,26 @@ The :mod:`runpy` module provides two functions:
    there is no such module at the specified location.
 
    The optional dictionary argument *init_globals* may be used to pre-populate
-   the module's globals dictionary before the code is executed. The supplied
+   the module globals dictionary before the code is executed. The supplied
```
Comment (KenanHArik) [inline Doc/library/runpy.rst]:
```suggestion
   the module's globals dictionary before the code is executed. The supplied
```

File: Doc/library/runpy.rst [hunk 2]
```diff
@@ -44,24 +44,24 @@ The :mod:`runpy` module provides two functions:
    returned.
 
    The optional dictionary argument *init_globals* may be used to pre-populate
-   the module's globals dictionary before the code is executed. The supplied
+   the module globals dictionary before the code is executed. The supplied
```
Comment (KenanHArik) [inline Doc/library/runpy.rst]:
```suggestion
   the module's globals dictionary before the code is executed. The supplied
```

**Comment by KenanHArik** [comment]
At PyCascades Sprint. I have reviewed the proposed language updates and added a few suggestions. Other than that it looks good to me. @willingc can you please review. Thanks!

File: Doc/library/runpy.rst (hunk 1 above)
Comment (geryogam) [reply Doc/library/runpy.rst]: Done.

Comment (maggyero) [reply Doc/library/runpy.rst]: Done.

**Comment by maggyero** [comment]
@KenanHArik Thanks for you reviewing this.

File: Doc/library/runpy.rst [hunk 3]
```diff
@@ -120,19 +120,19 @@ The :mod:`runpy` module provides two functions:
...
+   dictionary before the module code is executed. (Note that this is a
    minimal set of variables - other variables may be set implicitly as an
-   interpreter implementation detail).
+   interpreter implementation detail.)
 
    ``__name__`` is set to *run_name* if this optional argument is not
    :const:`None` and to ``'<run_path>'`` otherwise.
 
-   If the supplied path directly references a script file (whether as source
+   If ``file_path`` directly references a script file (whether as source
```
Comment (matrixise) [inline Doc/library/runpy.rst]: Hi @maggyero  at line 27, there is an other reference to `supplied path`. 
Thank you

**Review by matrixise — DISMISSED** [review]
Thank you for your proposal

File: Doc/library/runpy.rst (hunk 3 above)
Comment (geryogam) [reply Doc/library/runpy.rst]: Thanks, I had missed several of these!

**Comment by maggyero** [comment]
I have made the requested changes; please review again.

File: Doc/library/runpy.rst [hunk 4]
```diff
@@ -32,36 +32,36 @@ The :mod:`runpy` module provides two functions:
    .. index::
       pair: module; __main__
 
-   Execute the code of the specified module and return the resulting module
-   globals dictionary. The module's code is first located using the standard
+   Execute the code of the specified module and return the resulting module's
+   globals dictionary. The module code is first located using the standard
```
Comment (erlend-aasland) [inline Doc/library/runpy.rst]: I'm not sure this edit is correct; I prefer to leave it out.

```suggestion
   globals dictionary. The module's code is first located using the standard
```

File: Doc/library/runpy.rst [hunk 5]
```diff
@@ -10,7 +10,7 @@
 
 --------------
 
-The :mod:`runpy` module is used to locate and run Python modules without
+The :mod:`runpy` module is used to locate and execute Python modules without
```
Comment (erlend-aasland) [inline Doc/library/runpy.rst]: "run" and "execute" can be used interchangeably. There is for example a collection of C API with the PyRun prefix (see the C API docs for more details). Changing "run" with "execute" adds no value, so better leave it out and avoid the churn.

```suggestion
The :mod:`runpy` module is used to locate and run Python modules without
```

File: Doc/library/runpy.rst [hunk 6]
```diff
@@ -90,7 +90,7 @@ The :mod:`runpy` module provides two functions:
    .. versionchanged:: 3.4
       Updated to take advantage of the module spec feature added by
       :pep:`451`. This allows ``__cached__`` to be set correctly for modules
-      run this way, as well as ensuring the real module name is always
+      executed this way, as well as ensuring the real module name is always
```
Comment (erlend-aasland) [inline Doc/library/runpy.rst]:
```suggestion
      run this way, as well as ensuring the real module name is always
```

File: Doc/library/runpy.rst [hunk 7]
```diff
@@ -32,36 +32,36 @@ The :mod:`runpy` module provides two functions:
...
-   globals dictionary. The module's code is first located using the standard
+   Execute the code of the specified module and return the resulting module's
+   globals dictionary. The module code is first located using the standard
    import mechanism (refer to :pep:`302` for details) and then executed in a
    fresh module namespace.
 
    The *mod_name* argument should be an absolute module name.
    If the module name refers to a package rather than a normal
-   module, then that package is imported and the :mod:`__main__` submodule within
+   module, then that package is imported and the :mod:`!__main__` submodule within
```
Comment (erlend-aasland) [inline Doc/library/runpy.rst]:
```suggestion
   module, then that package is imported and the :mod:`__main__` submodule within
```

File: Doc/library/runpy.rst [hunk 8]
```diff
@@ -104,11 +104,11 @@ The :mod:`runpy` module provides two functions:
...
 
    Execute the code at the named filesystem location and return the resulting
-   module globals dictionary. As with a script name supplied to the CPython
-   command line, the supplied path may refer to a Python source file, a
+   module's globals dictionary. As with a script name supplied to the CPython
+   command line, *file_path* may refer to a Python source file, a
    compiled bytecode file or a valid :data:`sys.path` entry containing a
-   :mod:`__main__` module
-   (e.g. a zipfile containing a top-level ``__main__.py`` file).
+   :mod:`!__main__` module
```
Comment (erlend-aasland) [inline Doc/library/runpy.rst]:
```suggestion
   :mod:`__main__` module
```

File: Doc/library/runpy.rst [hunk 9]
```diff
@@ -119,34 +119,34 @@ The :mod:`runpy` module provides two functions:
...
-   supplied path, and ``__spec__``, ``__cached__``, ``__loader__`` and
+   If *file_path* directly references a script file (whether as source
+   or as precompiled byte code), then ``__file__`` will be set to
+   *file_path*, and ``__spec__``, ``__cached__``, ``__loader__`` and
    ``__package__`` will all be set to :const:`None`.
 
-   If the supplied path is a reference to a valid :data:`sys.path` entry, then
-   ``__spec__`` will be set appropriately for the imported :mod:`__main__`
+   If *file_path* is a reference to a valid :data:`sys.path` entry, then
+   ``__spec__`` will be set appropriately for the imported :mod:`!__main__`
```
Comment (erlend-aasland) [inline Doc/library/runpy.rst]:
```suggestion
   ``__spec__`` will be set appropriately for the imported :mod:`__main__`
```

**Review by erlend-aasland — APPROVED** [review]

**Pull request merged**